src/pages/AccountPage.js

```diff
@@ -15,13 +17,33 @@ import {
 import Avatar from '../components/Account/Avatar';
 import LoginManagement from '../components/Account/LoginManagement';
 import PasswordChangeForm from '../components/PasswordChange/PasswordChange';
+import SearchResults from '../components/Books/SearchResults';
+
+import {index} from '../components/Algolia';
 
 const AccountPage = props => {
+  const onSearchBooks = uid => {
+    index
+      .search({
+        query: uid
+      })
+      .then(res => {
+        props.storeBooks(res.hits);
+      })
+      .catch(err => {
+        console.error(err);
+      });
+  };
+
+  useEffect(() => {
```

Comment:
On the pageload, the users books will be fetched into the store and displayed on the page.